frontend/src/routes/candidate/(protected)/questions/all/+page.svelte

```diff
@@ -0,0 +1,121 @@
+
+  function gotoToQuestion(id: string) {
+    const questionUrl = $page.url.pathname.replace(/\/all$/, '/' + id);
+    setTimeout(() => goto(questionUrl), DELAY_M_MS);
+  }
+</script>
+
+<BasicPage title={$_('candidateApp.allQuestions.title')}>
+  <svelte:fragment slot="note">
+    <div class="text-warning">
+      <Icon name="important" />
+      {$_('candidateApp.allQuestions.warning', {values: {nofUnasweredQuestions}})}
+    </div>
+  </svelte:fragment>
+
+  <p class="text-center">
+    {$_('candidateApp.allQuestions.info')}
+  </p>
+
+  <p>
+    {#each categories as category}
+      <div class="edgetoedge-x">
+        <Expander title={category || ''} variant="category">
+          {#each questions as question, i}
+            <!-- Show questions based on categories -->
+            {#if category === question.category}
+              <!-- Question has been answered -->
+              {#if answerStore[question.id]}
+                <div class="pb-20 pt-20">
+                  <div class="text-accent">
+                    {question.category}
+                  </div>
+
+                  <Expander title={question.text || ''} variant="question">
+                    {question.info}
+                  </Expander>
+
+                  <div class="pt-10">
+                    <!-- This gives empty form label error for every empty dot, but fix should come from LikertResponseButton -->
+                    <LikertResponseButtons
+                      name={question.id}
+                      mode="display"
+                      options={question.options}
+                      selectedKey={answerStore[question.id].key} />
+
+                    {#if answerStore[question.id].openAnswer}
+                      <div class="pt-10">
+                        <label for="openAnswer{i}" class="text-m uppercase"
+                          >{$_('candidateApp.allQuestions.commentOnThisIssue')}
+                        </label>
+                        <textarea
+                          bind:value={answerStore[question.id].openAnswer}
+                          disabled={true}
+                          id="openAnswer{i}"
+                          rows="4"
+                          class="textarea textarea-primary w-full" />
+                      </div>
+                    {/if}
+                  </div>
+                </div>
+                <!-- Question not yet answered -->
+              {:else}
+                <div class="pt-30 pb-20">
+                  <div class="text-accent">
+                    {question.category}
+                  </div>
+
+                  <Expander
+                    title={question.text || ''}
+                    variant="question"
+                    titleClass="text-warning">
+                    {question.info}
+                  </Expander>
+
+                  <div class="flex justify-center pt-10">
+                    <Button
```

Comment:
Instead of navigating to the next page programatically using a Button, we should have a link. This, for example, allows users to control click the button in order to open it in a new tab. I think you can just add a Link element around the button and put the pathname in the href